codec-http2/src/test/java/io/netty/handler/codec/http2/Http2FrameCodecTest.java

```diff
@@ -227,7 +230,7 @@ public void sendGoAway() throws Exception {
         Http2GoAwayFrame goAwayFrame = new DefaultHttp2GoAwayFrame(Http2Error.NO_ERROR.code(), debugData.slice());
         goAwayFrame.setExtraStreamIds(2);
 
-        inboundHandler.writeOutbound(releaseLater(goAwayFrame));
+        inboundHandler.writeOutbound(goAwayFrame);
```

Comment:
Agreed. However `debugData` is used after this point to verify test conditions and it is possible it may be released, right? Do we need to retain `debugData` to ensure it is usable for test verification (or copy to `byte[]`)?